admin/templates/configuration/etc/init.d/mysql-proxy

```diff
@@ -77,6 +86,7 @@ start_cmd() {
     start -p $pidfile -l $lockfile $NAME "${cmd}"
     ret=$?
     [ $ret -eq 0 ] || echo "See startup logfile : ${logfile}"
+    prime_qserv
```

Comment:
I can imagine that this could fail occasionally if proxy is slow to start for some reason or if it does not bind mysql socket immediately.

Reply:
It's an ugly hack, I agree. I'll be happy to see it go, but this ticket's scope is much too big already, and poking around the proxy to get qserv to start right away can wait a little. I don't think start returns until after the proxy is really running.

I don't think this hack is really needed at this point, but its existence encourages someone (anyone) to change the proxy. If there's consensus, I'll remove it before merging.